### DISCUSSION TITLE
How to set a custom DNS server? and in general how to change the runner config

### DISCUSSION THREAD

**Discussion by lefuturiste** [firecow/gitlab-ci-local#830] [General]
Hello, I have a job with some command in the `script: ` that make use of network hosts that can only be resolved with a specific DNS server.
In normal time I have no issues resolving these specifics hosts during docker build or during docker run because I have modified my `/etc/docker/daemon.json` to make it my default DNS server.
With gitlab-ci-local when I launch my job the program inside the script report a `[Errno -2] Name or service not known` and fail.
So how can I configure this. 
I tried to look and there is no obvious way to change that.
May be there is a way to append some custom args to the  `dockerCmd` variable used in `src/job.ts` ?

**Comment by firecow** [comment]
How would you go about doing this on a "remote" runner?

Can you provide an example, we can use to create a feature request from?
